sdk/monitor/monitor-opentelemetry-exporter/samples-dev/httpClient.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * OpenTelemetry HTTPS Instrumentation allows the user to 
+ * automatically collect trace data and export them to 
+ * the backend of choice (we can use Zipkin or Jaeger for this example), 
+ * to give observability to distributed systems.
+ * 
+ * This is a simple example that demonstrates tracing HTTPS request from client to server.
+ * The example shows key aspects of tracing such as
+ * - Root Span (on Client)
+ * - Child Span (on Client)
+ * - Child Span from a Remote Parent (on Server)
+ * - SpanContext Propagation (from Client to Server)
+ * - Span Events
+ * - Span Attributes
+ *
+ * @summary OpenTelemetry https Instrumentation
```

Comment:
```suggestion
 * @summary demonstrates OpenTelemetry https Instrumentation
```

Is this specifically about how OTel will instrument the built-in`"https"` module for you? I would make that clear (about being the _built-in_ HTTPS module instead of just HTTP/S in general).